Erreur de nom : vous semblez utiliser un identifiant qui n'existe pas. Auriez-vous fait une typo ?
NameError("global name 'rback' is not defined",)
  File "<script>", line 18, in <module>
  File "<script>", line 11, in haveBaggle
